### PR TITLE
1.11.3 RN

### PR DESCRIPTION
**Pull request by anithamuthu79**
## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by orsenthil** [comment]
Please add the IPv6 update too. - `DCOS-34536 - Fixed DC/OS upgrade issue with ipv6 address configuration.`  I will approve again.

**Comment by anithamuthu79** [comment]
@orsenthil @cneth - I've added the DCOS-34536 entry.

File: pages/1.11/release-notes/1.11.3/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.3
+title: Release Notes for 1.11.3
+menuWeight: 0
+excerpt: Release notes for DC/OS 1.11.3
+---
+
+DC/OS 1.11.3 is released on June 26, 2018.
```
Comment (masonse) [inline pages/1.11/release-notes/1.11.3/index.md]: *was* released.

Comment (anithamuthu79) [reply pages/1.11/release-notes/1.11.3/index.md]: @seth I'll change the "tense (is >> was) after the release is done.

**Comment by anithamuthu79** [comment]
@judithpatudith could you review and share feedback/approve this PR.

**Comment by anithamuthu79** [comment]
@somikbehera could you review and share feedback/approve this PR.

File: pages/1.11/release-notes/1.11.3/index.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+- DCOS-22458 - Improved stability of dcos-checks-poststart-service-unhealthy check. [enterprise type="inline" size="small" /]
+- DCOS-25602 - DC/OS IAM: Reduced excessive logging which was accidentally introduced in 1.11.1 release. [enterprise type="inline" size="small" /]
+- DCOS-34435 - DC/OS Cosmos: Increased the limit of max-payload size at /v2/apps Marathon end point.
+- DCOS-34536 - Fixed DC/OS upgrade issue with ipv6 address configuration.
+- DCOS-38015 - Enhanced Mesos TLS cipher suites support with secure cipher suites. [enterprise type="inline" size="small" /]
+
+# Notable Changes in DC/OS 1.11.3
+
+- DCOS-19427 - CockroachDB: Set cluster version to 1.1. [enterprise type="inline" size="small" /]
+- DCOS_OSS-2417 and DCOS_OSS-3548 - Support for CoreOS 1688.4.0 and CoreOS 1688.5.3.
```
Comment (gauripowale) [inline pages/1.11/release-notes/1.11.3/index.md]: For lines 35 & 40 CoreOS 1688.4.0 should be removed as it was not tested for 1.11.3, only CoreOS 1688.5.3 was tested & verified.

Comment (anithamuthu79) [reply pages/1.11/release-notes/1.11.3/index.md]: As per Gauri's test results, @orsenthil could you please remove CoreOS 1688.4.0 info from https://github.com/dcos/dcos/blob/1.11.3/CHANGES.md

File: pages/1.11/release-notes/1.11.3/index.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+- DCOS-11714 - DC/OS IAM: Added support for anonymous bind via LDAP. [enterprise type="inline" size="small" /]
+- DCOS-22458 - Improved stability of dcos-checks-poststart-service-unhealthy check. [enterprise type="inline" size="small" /]
+- DCOS-25602 - DC/OS IAM: Reduced excessive logging which was accidentally introduced in 1.11.1 release. [enterprise type="inline" size="small" /]
+- DCOS-34435 - DC/OS Cosmos: Increased the limit of max-payload size at /v2/apps Marathon end point.
+- DCOS-34536 - Fixed DC/OS upgrade issue with ipv6 address configuration.
+- DCOS-38015 - Enhanced Mesos TLS cipher suites support with secure cipher suites. [enterprise type="inline" size="small" /]
+
+# Notable Changes in DC/OS 1.11.3
+
+- DCOS-19427 - CockroachDB: Set cluster version to 1.1. [enterprise type="inline" size="small" /]
```
Comment (jgehrcke) [inline pages/1.11/release-notes/1.11.3/index.md]: This is an internal change which should not be mentioned here.

File: pages/1.11/release-notes/1.11.3/index.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+- Marathon 1.6.496 [change log](https://github.com/dcos/dcos/blob/1.11.3/packages/marathon/buildinfo.json).
+- Metronome 0.4.2 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.2).
+
+
+# Issues Fixed in DC/OS 1.11.3
+
+- COPS-3158 - The check in custom installer for ftype=1 on Mesos and Docker work directories is added only when using XFS.
+- DCOS-11714 - DC/OS IAM: Added support for anonymous bind via LDAP. [enterprise type="inline" size="small" /]
+- DCOS-22458 - Improved stability of dcos-checks-poststart-service-unhealthy check. [enterprise type="inline" size="small" /]
+- DCOS-25602 - DC/OS IAM: Reduced excessive logging which was accidentally introduced in 1.11.1 release. [enterprise type="inline" size="small" /]
```
Comment (jgehrcke) [inline pages/1.11/release-notes/1.11.3/index.md]: This is an internal change that should not be mentioned here.

File: pages/1.11/release-notes/1.11.3/index.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+- DCOS-34435 - DC/OS Cosmos: Increased the limit of max-payload size at /v2/apps Marathon end point.
+- DCOS-34536 - Fixed DC/OS upgrade issue with ipv6 address configuration.
+- DCOS-38015 - Enhanced Mesos TLS cipher suites support with secure cipher suites. [enterprise type="inline" size="small" /]
```
Comment (jgehrcke) [inline pages/1.11/release-notes/1.11.3/index.md]: This is an internal change which should not be mentioned here.

File: pages/1.11/release-notes/1.11.3/index.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.3/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.11.3 includes the following:
+
+- Apache Mesos 1.5.2 [change log](https://github.com/mesosphere/mesos/blob/197c9105c2ed08f1529602f1ff72de846cde98c6/CHANGELOG).
+- Marathon 1.6.496 [change log](https://github.com/dcos/dcos/blob/1.11.3/packages/marathon/buildinfo.json).
+- Metronome 0.4.2 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.2).
```
Comment (jgehrcke) [inline pages/1.11/release-notes/1.11.3/index.md]: This section does not make clear if these were changed compared to 1.11.2. Why did we change the structure that we previously used?

File: pages/1.11/release-notes/1.11.3/index.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+
+# Issues Fixed in DC/OS 1.11.3
+
+- COPS-3158 - The check in custom installer for ftype=1 on Mesos and Docker work directories is added only when using XFS.
```
Comment (jgehrcke) [inline pages/1.11/release-notes/1.11.3/index.md]: It's really hard to understand what that means.

File: pages/1.11/release-notes/1.11.3/index.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+- DCOS-11714 - DC/OS IAM: Added support for anonymous bind via LDAP. [enterprise type="inline" size="small" /]
+- DCOS-22458 - Improved stability of dcos-checks-poststart-service-unhealthy check. [enterprise type="inline" size="small" /]
```
Comment (jgehrcke) [inline pages/1.11/release-notes/1.11.3/index.md]: What does this mean from the user's point of view?

**Pull request merged**